fortran90.html.markdown

```diff
@@ -436,6 +436,36 @@ contains
 
 end module fruity
 
+
+! ISO Standard Fortran 2008 introduced the DO CONCURRENT construct to allow you
+! to express loop-level parallelis
```

Comment:
Typo: parallelism

Reply:
@vendethiel Amended in #4878

Reply:
that was to the wrong branch on the main repo